examples/models/widgets.py

```diff
@@ -39,15 +39,15 @@
 
 radio_button_group = RadioButtonGroup(labels=["Option 1", "Option 2", "Option 3"], active=0)
 
-text_input = TextInput(placeholder="Enter value ...")
+text_input = TextInput(placeholder="Enter value ...", title="<b>text</b>")
```

Comment:
Users may depend on the fact that `title` used to accept raw strings. This is a breaking change. A bigger concern, however, is mixing raw strings with HTML, which never is a good idea (refer to PHP and a ton of other language/frameworks to see how this works out). Raw strings and HTML have to be separated to avoid user confusion, subtle bugs and security concerns. Unfortunately, we already broke this rule in a few places ourselves. We need to fix this across bokeh. Here, I would go with `TextInput(..., title=HTML("<b>text</b>"))`, where `HTML` is a model or a struct (raw `HasProps`), and leave the default for `title` to be raw string (so the type would be `title = Either(String, HTML)`).